projects/storefrontlib/src/shared/components/form/file-upload/file-upload.component.html

```diff
@@ -0,0 +1,25 @@
+<input
+  type="file"
+  [formControl]="control"
+  [accept]="allowedExtensions"
+  (change)="selectFile($event.target.files[0])"
+  #uploadCSV
```

Comment:
If we want to have a generic component for upload we should avoid specific name convention for formats. So hope that `#uploadCSV` can be changed for some more generic.